ui/components/component-library/text-field/text-field.scss

```diff
@@ -17,8 +17,7 @@
   border-color: var(--color-border-default);
 
   &--focused {
-    outline: 5px auto Highlight; // firefox
-    outline: 5px auto -webkit-focus-ring-color; // chrome
+    border: 2px var(--color-primary-default);
```

Comment:
I think this looks good but have we considered the accessibility impact of these changes? It will also effect the error state because we will not be able to have a focused and error state

<img width="501" alt="Screenshot 2025-04-22 at 12 40 51 PM" src="https://github.com/user-attachments/assets/d334163c-6835-4df8-ba1b-38869a3f82f1" />

We may be able to achieve the same effect but using outline. We could also make it a global style by adding it to `ui/css/base-styles.scss`

```scss
// Focus styles for all interactive components
:focus:not(:focus-visible) {
  outline: none;
}

:focus-visible {
  outline: 2px solid var(--focus-highlight-color);
  outline-offset: 2px;
  transition: outline-offset 0.2s ease-in-out;
}
```